test/templates/spec/repl_console_spec.js

```diff
@@ -0,0 +1,8 @@
+var assert = chai.assert;
+
+describe("REPLConsole", function() {
+
+  it("pending");
```

Comment:
No need to commit pending tests, commit them once you have them. We can merge this PR even without them and do them in a separate one.